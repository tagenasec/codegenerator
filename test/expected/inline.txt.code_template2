// inline template: data file: "test/workdir/inline.yaml"
{{- range $listItem := $.listOfItems }}
List Item: {{ $listItem }}
List Item quoted: {{ $listItem | quote }}
List Item capitalCamelCase: {{ $listItem | capitalCamelCase }}
{{- end }}
// inline template end
