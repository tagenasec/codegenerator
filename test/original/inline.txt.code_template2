// inline template: data file: "test/workdir/inline.yaml"
capitalCamelCase:
{{- range $listItem := $.listOfItems }}
List Item: {{ $listItem }}
List Item quoted: {{ $listItem | quote }}
List Item capitalCamelCase: {{ $listItem | capitalCamelCase }}
{{- end }}
orVoid:
{{ $.block.doesNotExist }}
{{ $.block.doesNotExist | orVoid }}
{{ $.block.aKey | orVoid }}
goType:
number: {{ "number" | goType }}
string: {{ "string" | goType }}
boolean: {{ "boolean" | goType }}
SomethingElse: {{ "SomethingElse" | goType }}
goTypeWithModule:
number: {{ "number" | goTypeWithModule "*model." }}
string: {{ "string" | goTypeWithModule "*model." }}
boolean: {{ "boolean" | goTypeWithModule "*model." }}
SomethingElse: {{ "SomethingElse" | goTypeWithModule "*model." }}
uppercaseToCapitalized:
PUT: {{ "PUT" | uppercaseToCapitalized }}
replace:
a/b/${id}: {{ "a/b/${id}" | replace "${id}" ":id" }}
// inline template end
