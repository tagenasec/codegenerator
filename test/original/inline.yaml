listOfItems:
  - firstItem
  - secondItem
block:
  aKey: aValue
