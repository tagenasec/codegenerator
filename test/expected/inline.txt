capitalCamelCase:
List Item: firstItem
List Item quoted: "firstItem"
List Item capitalCamelCase: FirstItem
List Item: secondItem
List Item quoted: "secondItem"
List Item capitalCamelCase: SecondItem
orVoid:
<no value>
void
aValue
goType:
number: int64
string: string
boolean: bool
SomethingElse: SomethingElse
goTypeWithModule:
number: int64
string: string
boolean: bool
SomethingElse: *model.SomethingElse
uppercaseToCapitalized:
PUT: Put
replace:
a/b/${id}: a/b/:id
